### DISCUSSION TITLE
About sessiongroup

### DISCUSSION THREAD

**Discussion by lifengyu2005** [alibaba/DeepRec#350] [General]
We use multi-threaded calls in the session run interface. The purpose of using seesiongroup is to increase the concurrency of external tasks. Is the concurrency support inside the session run not good?

I have read the TF code and have some confusion. I hope to have in-depth communication. Thank you very much

**Comment by liutongxuan** [comment]
Please raise the question through DeepRec DingTalk group. thanks.

**Reply by lifengyu2005** [reply]
About dingtalk, what is the group number? How to join? I looked for deeprec and couldn't find it

**Reply by liutongxuan** [reply]
34375411

**Reply by sinomiko** [reply]
group member is up to limit
